lib/survey-gizmo-ruby.rb

```diff
@@ -17,4 +17,5 @@
 
 path = File.join(File.expand_path(File.dirname(__FILE__)), 'survey_gizmo')
 Dir["#{path}/*.rb"].each { |f| require f }
-Dir["#{path}/**/*.rb"].each { |f| require f }
```

Comment:
Here we need to keep the `**/*.rb` otherwise the `SurveyGizmo::API` won't be required and you wont be able to use thing like: `SurveyGizmo::API::Survey.all(all_pages: true)`

You can give it a try with an new rails project:

```
rails new -d sqlite3 test-survey-gizmo-rails
```

and in the Gemfile point at your local version of the gem

```
gem 'survey-gizmo-ruby' #, path: '../survey-gizmo-ruby'
```

Then if you do `bundle exec rails c` you'll have the following error:

```
ylecuyer@inwin:~/Projects/test-survey-gizmo-rails$ bundle exec rails c
Running via Spring preloader in process 7319
Loading development environment (Rails 6.1.5)
irb(main):001:0> SurveyGizmo::API::Survey.all(all_pages: true)
Traceback (most recent call last):
        1: from (irb):1
NameError (uninitialized constant SurveyGizmo::API)
irb(main):002:0> 
```

Instead of:

```
ylecuyer@inwin:~/Projects/test-survey-gizmo-rails$ bundle exec rails c
Running via Spring preloader in process 7463
Loading development environment (Rails 6.1.5)
irb(main):001:0> SurveyGizmo::API::Survey.all(all_pages: true)
Traceback (most recent call last):
        1: from (irb):1
RuntimeError (Not configured!)
```

which is expected